.github/workflows/build-native.yml

```diff
@@ -51,9 +51,11 @@ jobs:
         if: startsWith(matrix.build, 'android')
         run: echo "ANDROID_NDK=$ANDROID_NDK_LATEST_HOME" >> $GITHUB_ENV
 
-      - name: Downgrade XCode for MacOS
-        if: matrix.build == 'macos'
-        run: sudo xcodes select 14.3.1
+      - name: Select XCode for MacOS
+        if: startsWith(inputs.runner, 'macos')
+        run: |
+          xcodes installed
```

Comment:
Why do we need this?

Reply:
We don't, but I find it useful to see what versions are available when looking at the output of the step